sched/adaptdl_sched/policy/pollux.py

```diff
@@ -40,6 +40,39 @@ def __init__(self):
         self._min_util = 0.35
         self._max_util = 0.65
 
+    def _allocate_job(self, job_info, node_info_list):
+        """
+        A simple strategy that find the first available node for a new job.
+
+        Arguments:
+            job_info (dict): single element dictionary of the job info
```

Comment:
Maybe better to take a job_key, job_info instead of a singleton dict so the semantics are clear from the arguments themselves